### ISSUE TITLE
Proxy Protocol v2 not working

### ISSUE THREAD

**Issue by youzi** [uNetworking/uWebSockets.js#885]
Request to [ProxyProtocol.js](https://github.com/uNetworking/uWebSockets.js/blob/master/examples/ProxyProtocol.js) example behind Cloudflare fails (just hangs).

Tested with:
- uNetworking/uWebSockets.js#v20.20.0 fails
- uNetworking/uWebSockets.js#v20.19.0 fails
- uNetworking/uWebSockets.js#v19.1.0 fails
- uNetworking/uWebSockets.js#v18.12.0 fails
- **uNetworking/uWebSockets.js#v18.11.0 works**

Is there a API change that I'm missing?

**Comment by uNetworkingAB** [comment]
Can you provide an example. Like a wireshark capture of the request or similar

**Comment by uNetworkingAB** [comment]
Yeah I can trigger this locally with haproxy running as frontend server. Will have a look, and it should have a test similar to the smoke test
